geode-redis/src/main/java/org/apache/geode/redis/internal/gfsh/RedisCommand.java

```diff
@@ -31,7 +31,7 @@
 public class RedisCommand extends GfshCommand {
   public static final String REDIS = "redis";
   public static final String REDIS__HELP =
-      "Commands related to the Redis API for Geode.";
+      "Commands related to the Geode API for Compatibility with Redis.";
   public static final String REDIS__ENABLE_UNSUPPORTED_COMMANDS = "enable-unsupported-commands";
   public static final String REDIS__ENABLE_UNSUPPORTED_COMMANDS__HELP =
       "Boolean value to determine "
```

Comment:
@ringles & @nonbinaryprogrammer Should we remove these?

Reply:
I decided to leave everything related to unsupported commands because that seemed outside of the scope of this ticket

Reply:
Sounds good!